src/libman/libman.csproj

```diff
@@ -0,0 +1,62 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should we call this project Microsoft.Web.LibraryManager.CLI.csproj instead ? Same for the test project.